tornjak-backend/pkg/agent/auth/keycloak.go

```diff
@@ -0,0 +1,131 @@
+package auth
```

Comment:
should have some unit tests

Reply:
I have put some in, but given we are trying to separate the auth logic soon, they are not super comprehensive - will likely make sense when we tackle that piece?